.changes/next-release/bugfix-AWSSDKforJava-3eb5168.json

```diff
@@ -0,0 +1,5 @@
+{
+    "category": "AWS SDK for Java", 
```

Comment:
I've categorizing fixes in the http clients with the name of the client, in netty's case i've been using "Netty NIO Async HTTP Client"

Reply:
Okie dokie.